Curtains/Curtains.Domain/Models/SetModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class SetModel
+    {
+        public int Id { get; set; }
+        public string Title { get; set; } = null!;
+        public string Description { get; set; }
+        public int CurtainsId { get; set; }
```

Comment:
if it's nullable type we probably need to mark them with a '?' like 'int?'

Reply:
Ok